task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -574,20 +575,21 @@ class ActivitySubprocess(WatchedSubprocess):
     @classmethod
     def start(  # type: ignore[override]
         cls,
-        path: str | os.PathLike[str],
+        dag_rel_path: str | os.PathLike[str],
         what: TaskInstance,
         client: Client,
         target: Callable[[], None] = _subprocess_main,
         logger: FilteringBoundLogger | None = None,
+        bundle_info=None,
```

Comment:
nit: should we move this up next to the path?

Reply:
welp, was somewhat of a slog, but done